### PR TITLE
M3C 12 cards

### PR DESCRIPTION
**Pull request by Simisays**
image
image
image
image
image

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/barrowgoyf.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ Z | RememberMilled$ True | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Hidden$ True | Origin$ Graveyard,Exile | Destination$ Hand | ChangeType$ Creature.IsRemembered | SelectPrompt$ You may select a creature card among the card milled this way  | Optional$ True | SelectPrompt$ You may select a creature card and/or a land card. | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:Z:TriggerCount$DamageAmount
+DeckHas:Ability$Mill|Graveyard
+DeckHints:Ability$Mill|Graveyard
+Oracle:Barrowgoyf's power is equal to the number of card types among cards in all graveyards and its toughness is equal to that number plus 1.\nWhenever Barrowgoyf deals combat damage to a player, you may mill that many cards. If you do, you may put a creature card from among them into your hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/barrowgoyf.txt:17]: incomplete

File: forge-gui/res/cardsfolder/upcoming/bismuth_mindrender.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Bismuth Mindrender
+ManaCost:3 B
+Types:Creature Eldrazi  
+PT:4/3
+K:Devoid
+K:Menace
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDigUntil | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, that player exiles cards from the top of their library until they exile a nonland card. You may cast that card by paying life equal to that spell's mana value rather than paying its mana cost.
+SVar:TrigDigUntil:DB$ DigUntil | Defined$ TriggeredTarget | Valid$ Card.nonLand | ValidDescription$ nonland card | FoundDestination$ Exile | RevealedDestination$ Exile | RememberFound$ True | RememberRevealed$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Defined$ Remembered | ValidZone$ Exile | Valid$ Card.nonLand+IsRemembered | ValidSA$ Spell | PlayCost$ PayLife<ConvertedManaCost> | RememberObjects$ Remembered | Optional$ True | ForgetPlayed$ True | SubAbility$ DBRestRandomOrder
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bismuth_mindrender.txt:9]: lots of wrong params + SVar here

File: forge-gui/res/cardsfolder/upcoming/mutated_cultist.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Name:Mutated Cultist
+ManaCost:2 B
+Types:Creature Eldrazi Horror
+PT:1/3
+K:Devoid
+K:Deathtouch
+T:Mode$ SpellCast | ValidCard$ Card.Self | Execute$ TrigRemove | TriggerDescription$ When you cast this spell, remove all counters from up to one target permanent or opponent. The next spell you cast this turn costs 1 less to cast for each counter removed this way.
+SVar:TrigRemove:DB$ RemoveCounter | ValidTgts$ Permanent,Opponent | TargetMin$ 0 | TargetMax$ 1 | Defined$ Targeted | CounterType$ All | TgtPrompt$ Select up to one target permanent or opponent | CounterNum$ All | RememberAmount$ True | SubAbility$ DBEffect
+SVar:X:Count$RememberedNumber
+SVar:DBEffect:DB$ Effect | StaticAbilities$ ReduceCost | Triggers$ TrigCastSpell | SetChosenNumber$ X | RememberObjects$ Remembered | SubAbility$ DBCleanup
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/mutated_cultist.txt:10]: I can't seem to have it remember the amount of counters removed 

Thoughts ?

File: forge-gui/res/cardsfolder/upcoming/jyoti_moag_ancient .txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Jyoti, Moag Ancient
+ManaCost:2 G U
+Types:Legendary Creature Elemental
+PT:2/4
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 1/1 green Forest Dryad land creature token for each time you've cast a commander from the command zone this game.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ g_1_1_forest_dryad | TokenOwner$ You | AILogic$ Always
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jyoti_moag_ancient .txt:6]: what's the point of this AILogic?

File: forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Razorfield Ripper 
+ManaCost:2 W
+Types:Artifact Creature Equipment Rhino
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self,Creature.EquippedBy | Execute$ TrigPutCounter | TriggerDescription$ Whenever CARDNAME or equipped creature attacks, you get {E}, then it gets +X/+X until end of turn, where X is the amount of {E} you have.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 1 | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | Defined$ TriggeredAttackerLKICopy | NumAtt$ +X | NumDef$ +X
+K:Reconfigure:2:::AlternateCost$ PayEnergy<3>
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt:8]: did you test this?

File: forge-gui/res/cardsfolder/upcoming/jyoti_moag_ancient .txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Jyoti, Moag Ancient
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jyoti_moag_ancient .txt:1]: bad filename

File: forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt (hunk 5 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt:8]: Hmm I didn't, I used Bloodthorn Flail's script as an example.  I guess this needs to be wired up first ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt:8]: I will see what can be done

File: forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+SVar:X:Count$YourCountersEnergy
+SVar:HasAttackEffect:TRUE
+Oracle:Whenever Razorfield Ripper or equipped creature attacks, you get {E}, then it gets +X/+X until end of turn, where X is the amount of {E} you have.\nReconfigure – Pay 2 or {E}{E}{E}.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt:11]: oracle incomplete

**Review by tool4ever — APPROVED** [review]

**Pull request merged**